src/components/BasicTable.jsx

```diff
@@ -43,36 +79,43 @@ function Cell({ row, column, rowIndex, columnIndex }) {
     );
 
   return (
-    <td className={['p-3', 'px-8', column.cellClasses].join(' ')}>{value}</td>
+    <td className={['p-3', 'px-8', 'text-wrap', column.cellClasses].join(' ')}>
+      {value}
+    </td>
   );
 }
 
-export function BasicTable({ columns, rows, options = {} }) {
+export function BasicTable({
+  columns,
+  rows,
+  getRowKey,
+  activeSort,
+  setActiveSort
+}) {
   return (
     <Card className="h-full w-full overflow-scroll">
```

Comment:
```suggestion
    <Card className="h-full w-full">
```
I don't think this will ever overflow, and overflow-scroll can add scrollbar gutters depending on browser/OS settings

Reply:
Thanks!